### PR TITLE
[UK] Add "Beatmap"

### PR DESCRIPTION
**Pull request by Aniterin**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmap/uk.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+- Мапи розроблені третіми лицями
+- Саморобні збірки які пересилаються через [торренти](https://en.wikipedia.org/wiki/Peer-to-peer).
+
+## Мапи та база гравців
+
+### Таблиці рекордів
+
+Гравці із всіх ігрових модів використовують мапи для [підняття ранку](/wiki/Performance_points) та [змагання з іншими](/wiki/Ranking). osu! має різні типи спеціальних таблиць, всі з яких, окрім першого доступні тільки [osu!supporter'ам](/wiki/osu!supporter):
+
+- Глобальна таблиця, який поширюється на всіх гравців які пройшли мапу;
```
Comment (endeshnik) [inline wiki/Beatmap/uk.md:92]:
```suggestion
- Глобальна таблиця, яка поширюється на гравців які пройшли мапу;
```

File: wiki/Beatmap/uk.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+- Глобальна таблиця, який поширюється на всіх гравців які пройшли мапу із [ігровими модифікаторами](/wiki/Game_modifier);
```
Comment (endeshnik) [inline wiki/Beatmap/uk.md:93]:
```suggestion
- Глобальна таблиця, яка поширюється на гравців які пройшли мапу з [ігровими модифікаторами](/wiki/Game_modifier);
```

**Review by endeshnik — CHANGES_REQUESTED** [review]
change мапи to карти

**Comment by Aniterin** [comment]
**Don't merge this yet, i will change some mistakes pointed by endeshnik**

**Comment by TicClick** [comment]
please fix broken links -- see https://github.com/ppy/osu-wiki/runs/7782457163?check_suite_focus=true

**Review by cl8n — APPROVED** [review]

**Pull request merged**